src/packager.c

```diff
@@ -450,6 +450,37 @@ _add_vba_project(lxw_packager *self)
     return LXW_NO_ERROR;
 }
 
+/*
+ * Write the xl/vbaProjectSignature.bin file.
+ */
+STATIC lxw_error
+_add_vba_project_signature(lxw_packager *self)
+{
+    lxw_workbook *workbook = self->workbook;
+    lxw_error err;
+    FILE *image_stream;
+
+    if (!workbook->vba_project_signature)
+        return LXW_NO_ERROR;
+    if (!workbook->vba_project)
+        return LXW_NO_ERROR;    // no VBA project: do not add signature
```

Comment:
C++ style comments aren't allowed/used for compatibility with older compilers. Just omit the comment.

Also, reverse the order of the checks (vba_project first) and leave one blank line between the if statements.

Reply:
Or merge the checks: `if (!workbook->vba_project || !workbook->vba_project_signature) ...`